support-frontend/assets/helpers/internationalisation/country.ts

```diff
@@ -642,175 +642,199 @@ const stripePaymentRequestAllowedCountries = [
 ];
 
 const countriesAffectedByVATStatus: IsoCountry[] = [
```

Comment:
24 countries were added, this is a big change because I have also ordered the list alphabetically